python/lsst/daf/butler/core/dimensions/_coordinate.py

```diff
@@ -156,6 +156,10 @@ class DataCoordinate(NamedKeyMapping[Dimension, DataIdValue]):
        between simple `dict` data IDs work, and hence using a `DataCoordinate`
        instance for at least one operand in any data ID comparison is strongly
        recommended.
+
+    See also
+    --------
+    :ref:`lsst.daf.butler-dimensions_data_ids`
```

Comment:
I tried building the docs but I didn't see this section included. May want to consider embedding it into the Notes section instead.

Reply:
It's there (or at least it is for me) - it's a little call-out box _above_ the Notes section, unlike the way it appears in the source.

Reply:
Ah. Yes. I see it now I know what I'm looking for. Sorry.